### PR TITLE
Parity: Bundle.allBundles/.allFrameworks

### PR DESCRIPTION
**Pull request by millenomi**
Implement this as a function of `CFBundleGetAllBundles()`. Add `.allFrameworks`, which is API that was missed.

Fixes https://bugs.swift.org/browse/SR-10360

### CODE REVIEW

**Comment by millenomi** [comment]
@swift-ci please test

File: CoreFoundation/PlugIn.subproj/CFBundle.c [hunk 1]
```diff
@@ -951,7 +951,13 @@ CF_PRIVATE CFURLRef _CFBundleCopyAppStoreReceiptURLInDirectory(CFURLRef bundleUR
 CFURLRef _CFBundleCopyAppStoreReceiptURL(CFBundleRef bundle) {
     return _CFBundleCopyAppStoreReceiptURLInDirectory(bundle->_url, bundle->_version);
 }
-        
+
+#ifdef CF_OPEN_SOURCE
```
Comment (compnerd) [inline CoreFoundation/PlugIn.subproj/CFBundle.c]: Who defines `CF_OPEN_SOURCE`?

Comment (millenomi) [reply CoreFoundation/PlugIn.subproj/CFBundle.c]: That's a stray #ifdef; it'll be removed shortly.

**Comment by millenomi** [comment]
@swift-ci please test

**Comment by millenomi** [comment]
@swift-ci please test

**Pull request merged**